Documentation/guides/AndroidAssetPacks.md

```diff
@@ -0,0 +1,176 @@
+.NET Android or .net Maui is not ideal. We need a simpler method.
+
+The new idea is to make use of additional metadata on `AndroidAsset`
+Items to allow the build system to split up the assets into packs
+automatically. So it is proposed that we implement support for something
+like this
+
+```xml
+<ItemGroup>
+   <AndroidAsset Include="Asset/data.xml" />
+   <AndroidAsset Include="Asset/movie.mp4" AssetPack="assets1" />
+   <AndroidAsset Include="Asset/movie2.mp4" AssetPack="assets1" />
+</ItemGroup>
+```
+
+In this case the additional `AssetPack` attribute is used to tell the
+build system which pack to place this asset in. Since auto import of items
+is common now we need a way for a user to add this additional attribute
+to auto included items. Fortunately we are able to use the following.
+
+```xml
+<ItemGroup>
+   <AndroidAsset Update="Asset/movie.mp4" AssetPack="assets1" />
+   <AndroidAsset Update="Asset/movie2.mp4" AssetPack="assets1" />
+   <AndroidAsset Update="Asset/movie3.mp4" AssetPack="assets2" />
+</ItemGroup>
+```
+
+This code uses the `Update` attribute to tell MSBuild that we are going
+to update a specific item. Note in the sample we do NOT need to include
+an `Update` for the `data.xml`, since this is auto imported it will still
+end up in the main feature in the aab.
+
+Additional attributes can be used to control what type of asset pack is
+produced. The only extra one supported at this time is `DeliveryType`,
+this can have a value of `InstallTime`, `FastFollow` or `OnDemand`.
+Additional attributes do not need to be included on ALL items. Any one
+will do, only the `AssetPack` attribute will be needed.
+See Google's [documentation](https://developer.android.com/guide/playcore/asset-delivery#asset-updates) for details on what each item does.
+
+```xml
+<ItemGroup>
+   <AndroidAsset Update="Asset/movie.mp4" AssetPack="assets1" DeliveryType="InstallTime" />
+   <AndroidAsset Update="Asset/movie2.mp4" AssetPack="assets1" />
+   <AndroidAsset Update="Asset/movie3.mp4" AssetPack="assets2" />
+</ItemGroup>
+```
+
+If the `AssetPack` attribute is not present, the default behavior will
+be to include the asset in the main application package.
+
+If however you have a large number of assets it might be more efficient to make use of the `base` asset pack setting. In this scenario you update ALL assets to be in a single asset pack then use the `AssetPack="base"` metadata to declare which specific assets end up in the base aab file. With this you can use wildcards to move most assets into the asset pack.
+
+```xml
+<ItemGroup>
+   <AndroidAsset Update="Assets/*" AssetPack="assets1" />
+   <AndroidAsset Update="Assets/movie.mp4" AssetPack="base" />
+   <AndroidAsset Update="Assets/some.png" AssetPack="base" />
+</ItemGroup>
+```
+
+In this example, `movie.mp4` and `some.png` will end up in the `base` aab file, but ALL the other assets
+will end up in the `assets1` asset pack.
+
+## Implementation Details
```

Comment:
It may be better to relegate this entire section to the commit message, and *not* have it in user-visible documentation.  I'm not sure it would be helpful for them.